core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -2954,7 +3012,7 @@ void wasmtime_ssp_proc_exit(
 __wasi_errno_t
 wasmtime_ssp_proc_raise(__wasi_signal_t sig)
 {
-    static const int signals[] = {
+    static const int signals[50] = {
```

Comment:
this change is not relevant to the PR (and is probably not valid either); could you revert it?